frontend/src/lib/components/CommandBar/ActionResults.tsx

```diff
@@ -0,0 +1,42 @@
+import { useValues } from 'kea'
+
+import { CommandResultDisplayable } from '../CommandPalette/commandPaletteLogic'
+
+import { actionBarLogic } from './actionBarLogic'
+import ActionResult from './ActionResult'
+import { getNameFromActionScope } from 'lib/components/CommandBar/utils'
+
+type ResultsGroupProps = {
+    scope: string
+    results: CommandResultDisplayable[]
+    activeResultIndex: number
+}
+
+const ResultsGroup = ({ scope, results, activeResultIndex }: ResultsGroupProps): JSX.Element => {
+    return (
+        <>
+            <div className="border-b pl-3 pr-2 pt-1 pb-1 bg-bg-3000-light">{getNameFromActionScope(scope)}</div>
+            {results.map((result) => (
+                <ActionResult
+                    key={`command_result_${result.index}`}
+                    result={result}
+                    focused={result.index === activeResultIndex}
+                />
+            ))}
+        </>
+    )
+}
+
+const ActionResults = (): JSX.Element => {
+    const { commandSearchResultsGrouped, activeResultIndex } = useValues(actionBarLogic)
+
+    return (
+        <div className="grow overscroll-none overflow-y-auto">
+            {commandSearchResultsGrouped.map(([scope, results]) => (
+                <ResultsGroup key={scope} scope={scope} results={results} activeResultIndex={activeResultIndex} />
+            ))}
+        </div>
+    )
+}
+
+export default ActionResults
```

Comment:
Personally I would like to not have any `export default` as it makes the auto-importing harder